src/bp/ui-admin/src/Pages/WorkspacePicker.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+import { Redirect } from 'react-router-dom'
+import { Card, CardBody } from 'reactstrap'
+
+import { setActiveWorkspace } from '../Auth'
+import logo from '../media/nobg_white.png'
+import api from '../api'
+
+class WorkspacePicker extends Component {
+  state = {
+    hasWorkspace: false
+  }
+
+  componentDidMount() {
+    this.loadWorkspaces()
+  }
+
+  async loadWorkspaces() {
+    const { data: workspaces } = await api.getSecured().get('/auth/me/workspaces')
+    if (workspaces.length) {
+      setActiveWorkspace(workspaces[0].workspace)
+      this.setState({ hasWorkspace: true })
+    }
+  }
+
+  render() {
+    if (this.state.hasWorkspace) {
+      return <Redirect to="/" />
+    }
+
+    return (
```

Comment:
what's that ? it seems like we don't allow user to select its workspace and we display a sorry message even if he has access to some ?

Reply:
Yep, we don't "support" them for now

Reply:
right gotcha